cmake/projects/OpenSSL/schemes/url_sha1_openssl.cmake.in

```diff
@@ -66,13 +66,23 @@ set(configure_command
 # Pass C flags through
 set(configure_opts ${configure_opts} ${CMAKE_C_FLAGS})
 
+# Pass BUILD_SHARED_LIBS (what about RPATH?)
+string(COMPARE EQUAL "@BUILD_SHARED_LIBS@" "ON" is_shared_lib)
```

Comment:
It does not seem to work with ```if (BUILD_SHARED_LIBS)```. My guess is we have to capture the variables with "@VARNAME@". Am I missing something?